app/assets/main/react_components/routes/dashboard/helpers/CategoryColorHelper.js

```diff
@@ -7,3 +7,23 @@ export const categoryName = (categories, itemID, isAchievement) => {
   }
   return 'unknown';
 };
+
+export const getCategoryColor = (category_tag) => {
+  if (category_tag == 'category_flights_active') {
+    return 'bg-blue-accent';
+  } else if (category_tag == 'category_unknown_active') {
+    return 'bg-primary';
+  } else if (category_tag == 'category_unknown') {
+    return 'bg-transparent';
+  } else if (category_tag == 'category_food_active') {
+    return 'bg-yellow-accent';
+  } else if (category_tag == 'category_public_emissions_active') {
+    return 'bg-gray-accent';
+  } else if (category_tag == 'category_car_active') {
+    return 'bg-orange-accent';
+  } else if (category_tag == 'category_shopping_active') {
+    return 'bg-pink-accent';
+  } else if (category_tag == 'category_housing_active') {
+    return 'bg-green-accent';
+  }
+};
```

Comment:
Dont we already have this in the css file?

Reply:
The group-hover attribute does not work with @apply, therefore I created a new file where only the color is returned and not the class.